.nvidia-ci.yml

```diff
@@ -180,7 +180,7 @@ sign:ngc:
 
 helm:ngc:
   image: ubuntu:latest
-  stage: release
```

Comment:
why is the stage changed from release to sign?, helm:ngc belongs to release stage.

Reply:
i don't get it, even with this change, i see pipeline showing same error. the order of stages is scan -> release-images -> sign-images -> helm-publish. Signing stage depends on release as expected, and helm should be on its own release-helm stage.